Packs/Armorblox/Integrations/Armorblox/Armorblox.py

```diff
@@ -77,9 +77,13 @@ def test_module(client: Client) -> str:
 
 
 def get_page_token(client, pageToken=None):
-    response = client.get_incidents(pageSize=MAX_INCIDENTS_TO_FETCH, pageToken=pageToken, first_fetch=FIRST_FETCH)
-    if 'next_page_token' in response.keys():
-        return response['next_page_token']
+    response, next_page_token = client.get_incidents(
+        pageSize=MAX_INCIDENTS_TO_FETCH,
+        pageToken=pageToken,
+        first_fetch=FIRST_FETCH
+    )
+    if next_page_token:
```

Comment:
This if-else doesn't seem useful - we can blindly return `next_page_token` as the default value is `None` anyway